praetorian_cli/sdk/test/ui/test_aegis_schedule_helpers.py

```
@@ -0,0 +1,393 @@
+
+# --- format_status tests ---
+
+class TestFormatStatus:
+    COLORS = {
+        'primary': 'cyan',
+        'accent': 'magenta',
+        'dim': 'dim',
+        'success': 'green',
+        'warning': 'yellow',
+        'error': 'red',
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)